rel/overlay/etc/default.ini

```diff
@@ -930,3 +930,99 @@ url = {{nouveau_url}}
 ;max_objects = 10000
 ;max_idle = 600000
 ;enable = true
+
+[couch_scanner]
+; How often to check for configuration changes and start/stop plugins
+;interval_sec = 5
+
+; Minimum time to force a plugin to wait before running again after a crash
+;min_penalty_sec = 30
+
+; Maximum time to force a plugin to wait after repeated crashes (8 hours default)
+;max_penalty_sec = 28800
+
+; If plugin runs successfully without crashing for this long, reset its
+; repeated error count
+;heal_threshold_sec = 300
+
+; Database processing rate limit per second. This will also be the
+; rate at which design documents are fetched. The rate is shared
+; across all running plugins.
+;db_rate_limit = 50
+
+; Limits the rate per second at which plugins may open db shard files
+; on a node. The rate is shared across all running plugins.
+;shard_rate_limit = 50
+
+; Limit the rate per second at which plugins open documents. The rate
+; is shared across all running plugins.
+;doc_rate_limit = 1000
+
+[couch_scanner_plugins]
+;couch_scanner_plugin_ddoc_features = false
+;couch_scanner_plugin_find = false
+
+; The following [$plugin*] settings apply to all plugins
+
+;[$plugin]
+; Run plugin on or after this time. The default is to run once after the
+; node starts. Times are in UTC. Possible time formats are:
+;  * Unix seconds: 1712338014
+;  * Date/Time: YYYY-MM-DDTHH, YYYY-MM-DDTHH:MM, YYYY-MM-DDTHH:MM:SS, YYYY-MM-DDTHH:MM:SSZ
+;after = restart
+; Run the plugin periodically. By default it will run once after node the node starts.
+; Possible period formats are:
+;  * $num_$timeunit: 1000_sec, 30_min, 8_hours, 24_hour, 2_days, 3_weeks, 1_month
+;  * $weekday: mon, monday, Thu, thursdays
+;repeat = restart
+
+;[$plugin.skips_dbs]
+; Skip over databases if their names contain any of the strings in this section.
+;string1 = true
+;string2 = true
```

Comment:
Wonder if instead of binary:compile / match we should make these regexes as well. They probably won't work as keys as they are restrictions so might be something like:

 ```
[$plugin.skip_dbs]
sometag1 = regex1
sometag2 = regex2
```

Tag won't matter much but would help make the shape look similar to the `"find"` plugin (could move the compile/match to util even?).

Or is the simpler contains string matcher better and just good enough for cases when we want to skip some dbs or doc_ids...

Reply:
Perhaps another possibility could be to just have a single "skip" section, and then entries like e.g.
```ini
[$plugin.skip]
db_regex = regex1
doc_regex = regex2
ddoc_regex = regex3
```
?
With such a design, I'm not sure if it's necessary for repeated keys, or if one could just construct more elaborate regexes using `|` etc.

Reply:
I was think of a case with a larger number of various dbs to skip and not having to build a long single string by hand concatenated with `|` and having to manage those with configuration management (concatenating and editing them). That's why though allowing multiple entries might be more ergonomic

For instance:

```ini
[$plugin.skip_dbs]
custdbs = customer_orders_
accounts = account_many_
testingstuff = test_data_foo_
random1 = _random_stuff_
```

Reply:
Yeah, for skipping a lot of different patterns, I agree this is nicer.

Reply:
I went using regexes only just to simplify things. This allowed the regex compiling and matching to be in the util and be shared.